app/components/Image/index.tsx

```diff
@@ -9,3 +9,12 @@ export const Image = (props: ImageProps) => {
   }
   return <NextImage {...props} />;
 };
+
+// FIXME: next/image doesn't export this interface so this is a temporary hack
```

Comment:
I think if you delete all your node modules, rebase latest staging, and reinstall, this error should go away.
Something changed when I removed /cms from our root workspace recently that required me to import directly.

If that works please revert this and the import changes, otherwise dm me and we can figure it out...